src/sentry/grouping/enhancer/__init__.py

```diff
@@ -228,6 +236,85 @@ def compare_rust_enhancers(
                 sentry_sdk.capture_message("Rust Enhancements mismatch")
 
 
+def assemble_rust_components(
```

Comment:
The coverage for this function and the next one is low. Could you please take a look into increasing it?

Reply:
👍🏻 we forgot to turn on the option for tests :-D